### DISCUSSION TITLE
January 2022 Updates

### DISCUSSION THREAD

**Discussion by brunoborges** [microsoft/openjdk#215] [Q&A] [answered]
The January 2022 updates of Microsoft Build of OpenJDK are now live at [microsoft.com/openjdk](https://microsoft.com/openjdk). 

This release includes new supported operating systems and architectures, as well as information regarding End of Life of Zulu for Azure binaries. Please [read the announcement](https://devblogs.microsoft.com/java/additions-and-updates-to-microsoft-build-of-openjdk/) for more details.

## OpenJDK 17 LTS ([download](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-17))

Release **17.0.2** now includes packages for the following operating systems:

- Linux (x64 / aarch64)
- Windows (x64 / aarch64)
- macOS (x64 / aarch64)
- Alpine Linux (x64; musl-libc)

## OpenJDK 11 LTS ([download](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11))

Release **11.0.14** now includes packages for the following operating systems:

- Linux (x64 / aarch64)
- Windows (x64 / aarch64)
- macOS (x64 / aarch64)
- Alpine Linux (x64; musl-libc)

_NOTE: The packages for macOS aarch64 are `Early Access` and may have not passed all tests._

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/openjdk/releases/tag/'>January 2022 Updates</a>.</em>

**Comment by karianna** [comment, marked as answer]
It was an oldie but a goodie!
